common/jenkins-slaves/nodejs10-angular/Dockerfile.rhel7

```diff
@@ -62,9 +62,14 @@ RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash - && \
     yum clean all -y
 
 COPY npmrc $HOME/.npm-global/etc/npmrc
+COPY npmrc $HOME/.npmrc
+COPY yarnrc $HOME/.yarnrc
 
 RUN sed -i "s|NEXUS_HOST|$NEXUS_HOST|g" $HOME/.npm-global/etc/npmrc && \
     sed -i "s|NEXUS_AUTH|$(echo -n $NEXUS_AUTH | base64)|g" $HOME/.npm-global/etc/npmrc && \
+    sed -i "s|NEXUS_HOST|$NEXUS_HOST|g" $HOME/.npmrc && \
+    sed -i "s|NEXUS_AUTH|$(echo -n $NEXUS_AUTH | base64)|g" $HOME/.npmrc && \
```

Comment:
proposal: instead of having two times this file/creation we could use symbolic linking:
```
ln -s $HOME/.npm-global/etc/npmrc $HOME/.npmrc
```